internal/flow/notify.go

```diff
@@ -44,15 +44,11 @@ func (s *Service) NotifyCommitter(ctx context.Context, event *http.PodNotifyRequ
 	if len(matches) < 2 {
 		return errors.WithMessagef(err, "locate service from commit message: '%s'", commit.Message)
 	}
-	env := matches[1]
+	// Use environment received from release-daemon
+	env := event.Environment
 	service := matches[2]
 	namespace := event.Namespace
-	// handle cases where the namespace would default to the environment
-	// this ensures that we can locate artifact specifications on deployments in
-	// these namespaces
-	if isEnvironmentNamespace(event.Namespace) {
```

Comment:
You can delete the function `isEnvironmentNamespace` as well now, as this was its only usage.